src/test/java/com/wikia/webdriver/testcases/adstests/TestVUAP.java

```diff
@@ -9,11 +10,18 @@
 
 public class TestVUAP extends TemplateNoFirstLoad {
   private static final Dimension DESKTOP_SIZE = new Dimension(1920, 1080);
+  private static final String TOP_LEADERBOARD_IMAGE = "TOP_LEADERBOARD";
+  private static final String BOTTOM_LEADERBOARD_IMAGE = "BOTTOM_LEADERBOARD";
+  private static final String FANDOM_URL = "http://www.wikia.com/fandom";
+
+  private final String currentBrowserTab() {
+    return driver.getWindowHandle();
+  }
 
   @Test(
           groups = "AdsUapDesktop"
   )
-  public void testVideoPlayFlow() {
+  public void adsTopAdVideoClosesWhenFinishPlaysOasis() {
     Page page = new Page("project43", "SyntheticTests/VUAP");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "SyntheticTests/VUAP" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "project43" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)